### ISSUE TITLE
Cannot index generics using tuple literal

### ISSUE THREAD

**Issue by layday** [microsoft/pyright#1260]
**Describe the bug**
In Python, `foo[bar]` is equivalent to `foo[(bar,)]`; the tuple parentheses can be omitted as in many other contexts.  Pyright is not able to parse generics with parenthesised tuples.

**To Reproduce**
Type check the snippet below.

**Expected behavior**
That `Union[(int, str)]` would be equivalent to `Union[int, str]` and would not generate an error.

**Screenshots or Code**

```python
# pyright: strict

from typing import Union

assert Union[int, str] == Union[(int, str)]

```

**VS Code extension or command-line**
Either, 1.1.94

**Comment by erictraut** [comment]
This will be addressed in the next release.

**Comment by layday** [comment]
(I don't know if this influenced the implementation, but I was incorrect that `foo[bar]` is equivalent to `foo[(bar,)]`; that is only true for comma-separated value indices.)

**Comment by erictraut** [comment]
This is now addressed in pyright 1.1.95, which I just published. It will also be included in the next release of pylance.
